gammapy/spectrum/models.py

```diff
@@ -733,6 +733,27 @@ def to_sherpa(self, name='default'):
 
         return model
 
+    @property
+    def e_peak(self):
+        """Peak energy of the spectral model.
```

Comment:
I would suggest:
```
Spectral energy distribution peak energy (`~astropy.utils.Quantity`).
```
Then you can delete the `Returns` section below to save some lines and duplicating the description what it is.

I think with "spectral energy distribution" most users will know that it's `e2dnde`, but just to be sure you could mention it explicitly like:
```
This is the peak in E^2 x dN/dE and is given by:
```